pkg/setting/setting.go

```diff
@@ -696,7 +696,7 @@ func (cfg *Cfg) loadSpecifiedConfigFile(configFile string, masterFile *ini.File)
 				defaultSec, _ = masterFile.NewSection(section.Name())
 			}
 			defaultKey, err := defaultSec.GetKey(key.Name())
```

Comment:
Alternatively, we could something like:

```
var defaultKey *ini.Key
if _, exists := defaultSec.KeysHash()[key.Name()]; !exists {
  defaultKey, _ = defaultSec.NewKey(key.Name(), key.Value())
} else {
  defaultKey, _ = defaultSec.GetKey(key.Name())	
}
```

PS: With proper error handling, I guess